ext/updater.go

```diff
@@ -163,24 +168,31 @@ func (u *Updater) StartPolling(b *gotgbot.Bot, opts *PollingOpts) error {
 	}
 
 	go u.Dispatcher.Start(b, bData.updateChan)
-	go u.pollingLoop(bData, reqOpts, v)
+	go u.pollingLoop(opts.ParentCtx, bData, reqOpts, v)
```

Comment:
note: this risks a nil pointer if opts is nil :)

Reply:
Shoot!